Makefile

```diff
@@ -14,3 +14,15 @@ clean:
 
 %.metallib: %.air
 	xcrun -sdk macosx metallib $< -o $@
+
+%.vert: test-data/%.wgsl $(wildcard src/*.rs src/**/*.rs examples/*.rs)
```

Comment:
I wonder if that wildcard can be put into a separate target, since otherwise we are repeating it too many times
(not required for landing this)